shell/platform/linux/BUILD.gn

```diff
@@ -156,7 +158,10 @@ executable("flutter_linux_unittests") {
 
   public_configs = [ "//flutter:config" ]
 
-  configs += [ "//flutter/shell/platform/linux/config:gtk" ]
+  configs += [
+    "//flutter/shell/platform/linux/config:gtk",
+    "//flutter/shell/platform/linux/config:wayland-egl"
```

Comment:
This should have a trailing comma and is stopping the build.

You can format this locally with `gn format shell/platform/linux/BUILD.gn`.